package.json

```diff
@@ -25,7 +32,7 @@
   "scripts": {
     "build": "tsc",
     "start": "tsc -w",
-    "prepare": "yarn build"
```

Comment:
Not sure why this was in here, all it does is slow down pipelines. Let me know if you disagree.